spec.emu

```diff
@@ -390,6 +407,31 @@ contributors: Google, Ecma International
     </emu-note>
   </emu-clause>
 
+  <emu-clause id="sup-temporal-calendardateextendedyear" type="abstract operation">
+    <h1>
+      CalendarDateExtendedYear (
+        _calendar_: a String,
+        _date_: a Temporal.PlainDateTime, Temporal.PlainDate, or Temporal.PlainYearMonth,
+      ): an integer
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>It performs implementation-defined processing to find the year for the date corresponding to _date_ in the context of the calendar represented by _calendar_ relative to a well-defined epoch year for that calendar.</dd>
+    </dl>
+    <p>
+        This definition supersedes the definition provided in "Temporal proposal", #sec-temporal-calendardateextendedyear.
+    </p>
+    <emu-alg>
+      1. Let _year_ be _date_.[[ISOYear]].
+      1. If IsCalendarSupportEra(_calendar_) is *false*, then
+        1. Return _year_.
+      1. Let _r_ be the row in <emu-xref href="#table-eras"></emu-xref> which _calendar_ is the value of the Calendar column as well as the Era column.
```

Comment:
After seeing this written out and seeing the table with some epoch year entries being blank, I think this  is kind of awkward, and I now understand your comment about it being per calendar and not per era more fully.

So I'm thinking maybe instead of a new column in the table as we had previously discussed, what about a separate table? What do you think?

Reply:
I prefer a separate table :D